client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/testdomain/protobuf/TestEntity.java

```diff
@@ -0,0 +1,55 @@
+package org.infinispan.client.hotrod.query.testdomain.protobuf;
+
+
+import java.io.IOException;
+
+import org.infinispan.protostream.MessageMarshaller;
+
+public class TestEntity {
+
+   private final String name1;
+   private final String name2;
+   private final String name3;
+   private final String name4;
+   private final String name5;
+   private final String name6;
+
+   public TestEntity(String name1, String name2, String name3, String name4, String name5, String name6) {
+      this.name1 = name1;
+      this.name2 = name2;
+      this.name3 = name3;
+      this.name4 = name4;
+      this.name5 = name5;
+      this.name6 = name6;
+   }
+
+   public static class TestEntityMarshaller implements MessageMarshaller<TestEntity> {
+
+      @Override
+      public TestEntity readFrom(ProtoStreamReader reader) throws IOException {
+         return new TestEntity(reader.readString("name1"), reader.readString("name2"),
```

Comment:
This lines makes a lot of pandas sad and depressed :)

Reply:
Each readString as a separate statement, so we do not give a bad example to others even if it just a test. But it's fine anyway :)